app-server/app/src/main/scala/user/Routes.scala

```diff
@@ -0,0 +1,48 @@
+package com.azavea.rf.user
+
+import scala.concurrent.ExecutionContext
+
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.PathMatchers.JavaUUID
+import akka.http.scaladsl.model.StatusCodes
+
+import com.azavea.rf.datamodel.latest.schema.tables.UsersRow
+
+import com.azavea.rf.utils.Database
+
+
+/**
+  * Routes for users
+  */
+trait UserRoutes {
+  def userRoutes()(implicit db:Database, ec:ExecutionContext) = (
+    pathPrefix("users") {
```

Comment:
No, I don't think so -- we usually haven't on other healthcheck endpoints and it isn't quite an API resource.